### ISSUE TITLE
Port Scanner Form Component in React

### ISSUE THREAD

**Issue by ishika1727** [vinitshahdeo/PortScanner#121]
As we've built many features to justify the one-liner description of this project i.e. a go-to networking tool, now it's time to revamp the existing UI.  

Add a new component `PortScannerForm.js` which will take inputs for:

- host to scan
- lower range of ports
- upper range of ports

This component should also have a button called `SCAN`. The scanning for open ports will start on clicking this button. Refer the image attached below for design

_As of now, on clicking `SCAN` button, the values in the input fields should be displayed in console_

P.S. : The basic frontend on react is available on `feature/frontend-react` branch in the folder `network-scanner`. Add your components in the same branch. Do not make any changes in `App.js` / `index.js`.  

![image](https://user-images.githubusercontent.com/42978414/110368399-fe87c280-806e-11eb-935f-c1beb5336b7f.png)

**Comment by devRawnie** [comment]
I would like to work on this. Please assign this to me.

**Comment by subhangi2731** [comment]
can you please assign it to me?

**Comment by vinitshahdeo** [comment]
Hello guys,
The branch will be merged soon. Review is still `in-progress`. Post that we'll assign the issues.

**Comment by vinitshahdeo** [comment]
@ishika1727 I have merged the PR. You can assign this issue with a story point of not more than 3 days.

**Comment by ishika1727** [comment]
@devRawnie Please go ahead with the issue and create a PR in `feature/react` branch by Monday EOD.

Feel free to contact me anytime

**Comment by devRawnie** [comment]
Sure thing. I am on it

**Comment by devRawnie** [comment]
Can I use reactstrap library to style the input fields?

**Comment by ishika1727** [comment]
@devRawnie yes, please go ahead, but try to keep it as simple as possible as we will be merging all components together after they have been made.

**Comment by devRawnie** [comment]
How do I add reviewers in the pull request, I couldn't figure it out while creating the pull request.

**Comment by ishika1727** [comment]
I ll add the reviewers, no issues

**Comment by ishika1727** [comment]
@devRawnie Merged your PR into `feature/react` branch. Amazing work. Please go through the other open issues as well and contribute more to this project
